plugins/modules/iam_user.py

```diff
@@ -380,9 +408,9 @@ def create_or_update_user(connection, module):
     user = get_user(connection, module, params['UserName'])
     if changed and new_login_profile:
         # `LoginProfile` is only returned on `create_login_profile` method
-        user['user']['password_reset_required'] = login_profile_data.get('LoginProfile', {}).get('PasswordResetRequired', False)
+        user['password_reset_required'] = login_profile_data.get('LoginProfile', {}).get('PasswordResetRequired', False)
 
-    module.exit_json(changed=changed, iam_user=user)
+    module.exit_json(changed=changed, user=user)
```

Comment:
This is a breaking change.  Is there a compelling reason to change what we return to users?

Reply:
I think it's what was meant to be returned all along - the updated data returned matched the documented RETURN section

Reply:
If we want to migrate it's probably better to return *both* but only document the new value.

Reply:
as it was originally, `user` was nested inside of `iam_user`, so to retrieve user attributes you had to call it by `iam_user.user.<attribute>`, which just feels off to me

Reply:
I agree that the old behaviour was "off", unfortunately this change will break existing playbooks, so we'll need to return the original values in addition to the new ones.

Reply:
okay - I will separate those changes and add a separate PR

Reply:
@tremble sorry, i misunderstood - added it as an extra value in addition to the original values. Do we know what the date of the deprecation should be? I just kept it the same as a reference I used